### PR TITLE
task 5 completed

### PR DESCRIPTION
**Pull request by Damz1**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const request = require("supertest");
 const db = require("../db/connection");
 const testData = require("../db/data/test-data");
 const seed = require("../db/seeds/seed");
+const { toBeSortedBy } = require("jest-sorted");
```
Comment (alexdevdavis) [inline __tests__/app.test.js:6]: is this a VSCode auto-add that can be destroyed now?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -96,3 +97,35 @@ describe("Get 200: /api/reviews/:review_id", () => {
...
+
+describe("Get 200: /api/reviews", () => {
+  test("should respond with an array of objects reviews with all properties", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const reviews = body.reviews;
+        expect(reviews).toBeInstanceOf(Array);
+        reviews.forEach((review) => {
```
Comment (alexdevdavis) [inline __tests__/app.test.js:109]: need a length check before a forEach, lest you get false positives

File: __tests__/app.test.js [hunk 3]
```diff
@@ -96,3 +97,35 @@ describe("Get 200: /api/reviews/:review_id", () => {
...
+          expect(review).toHaveProperty("owner");
+          expect(review).toHaveProperty("title");
+          expect(review).toHaveProperty("review_id");
+          expect(review).toHaveProperty("category");
+          expect(review).toHaveProperty("review_img_url");
+          expect(review).toHaveProperty("created_at");
+          expect(review).toHaveProperty("votes");
+          expect(review).toHaveProperty("designer");
+          expect(review).toHaveProperty("comment_count");
```
Comment (alexdevdavis) [inline __tests__/app.test.js:118]: test the expected data type of each property too

File: package.json [hunk 4]
```diff
@@ -29,12 +30,14 @@
     "husky": "^8.0.2",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
+    "jest-sorted": "^1.0.14",
     "pg-format": "^1.0.4",
```
Comment (alexdevdavis) [inline package.json:34]: ensure that `pg-format` is in `dependencies`, not `devDependencies`, lest your app break on deploy, when dev dependencies get pruned

**Review by alexdevdavis — CHANGES_REQUESTED** [review]
A couple of changes to make before PR is approved, as per comments. 
Great progress today 👍

File: __tests__/app.test.js (hunk 2 above)
Comment (Damz1) [reply __tests__/app.test.js:109]: done, but not sure why, can't it fail otherwise?

**Pull request merged**